crates/cli/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "cli"
```

Comment:
this should be `schematools-cli` so it can be published on crates.io

Reply:
Fixed. But i think we should only publish schematools on crates.io. CLI package is only for binary builds.

Reply:
I am less concerned about the CLI now, but previously we used it, and it was harder to sell to managers that we're installing it from git, which is a devsecops nightmare.  Also installing binaries from crate.io is quicker, and even quicker when combined with https://github.com/cargo-bins/cargo-binstall / https://github.com/cargo-bins/cargo-quickinstall